### ISSUE TITLE
Can't select menu when pressing start

### ISSUE THREAD

**Issue by masterrite** [suicvne/SpaceCadetPinball_Vita#3]
Menu opens automatically upon the first pressing of start button. Subsequent pressings of start only shows the menu bar, but none of the dropdowns can be selected.

**Comment by suicvne** [comment]
Yes this is a known issue with ImGui. ImGui does not seem to like focusing controls. I need to find a good workaround for it
